crates/sui-benchmark/src/bin/stress.rs

```diff
@@ -325,6 +328,9 @@ async fn main() -> Result<()> {
                     }
                 }
                 join_all(follower_handles).await;
+                loop {
```

Comment:
Could we add a comment about why this is necessary? Thanks!

Reply:
Added. This is to keep Sui validators alive.